### ISSUE TITLE
Reproducibility scores

### ISSUE THREAD

**Issue by MariamHossam00** [CXH-Research/DeVigNet#2]
Hello, thanks for your effort in this work.
I was interested in replicating your work, but unfortunately the scores I get are different from those in the paper after training for 200 epochs. 
Here is the config I am using for training and testing on 512x512 images:
```
OPTIM:
  BATCH_SIZE: 1
  NUM_EPOCHS: 200
  LR_INITIAL: 1e-4
  LR_MIN: 1e-6
  SEED: 3407
  WANDB: False

TRAINING:
  VAL_AFTER_EVERY: 1
  RESUME: False
  PS_W: 512
  PS_H: 512
  TRAIN_DIR: '../dataset/VigSet/train/' # path to training data
  VAL_DIR: '../dataset/VigSet/test/'     # path to validation data
  SAVE_DIR: './checkpoints/'          # path to save models and images
  ORI: False
```
I am using a fixed learning rate without scheduler, as it was not mentioned in the paper that you used one. The results I got are:

> PSNR = 21.99, SSIM = 0.74, MAE = 17.92, LPIPS = 0.199

I also have 2 more questions:

1. How many epochs did you train?
2. Did you use a different loss function other than the one in the paper (and also in the code)? This [comment](https://github.com/CXH-Research/DeVigNet/issues/1#issuecomment-2267995780) mentions that to get a similar LPIPS score as in the paper, we need to modify the loss function. However, the loss function mentioned in the paper did not have a term for LPIPS. So, I was wondering if you did an additional step that was not mentioned in the paper or if I am getting something wrong.

**Comment by xuhangc** [comment]
Hi there, thanks for the question.

1. Since the student who led these specific experiments has graduated, I don't have the exact training logs immediately at hand. However, based on our configuration, the model was likely trained for more than 300 epochs to ensure full convergence. I would recommend monitoring your validation metrics to determine the optimal stopping point for your specific dataset.
2. That is a keen observation. You are correct—while the core objective function is described in the paper, we found that explicitly optimizing for perceptual quality is crucial for reproducing the reported LPIPS scores. If you are training from scratch, it is indeed much harder to achieve the same perceptual results without it. I highly suggest adding an LPIPS loss term with a small weight (e.g., lambda_lpips = 0.1 or similar) to your total loss. This acts as a perceptual guide and stabilizes the training significantly.
